### PR TITLE
[Sage Only] Ports /tg/ Kilo Station

### PR DESCRIPTION
**Pull request by Cenrus**
## About this Pull Request

Adds /tg/s version of Kilo Station into MRP map rotation. Has a whiteship dock, and I have done the wiring from scratch and added the Cryopod room back.

## Why it's good for the game

qwerty told me to PR this, but I do not know how to prevent the voting of Kilo Station on golden

Many players on sage have been wanting Kilo to return as a low pop competitor for box. 

:cl: Okand37 and Cenrus
add: Kilo Station has returned to active map rotation on BeeStation Sage
/:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
I feel like if it's fixed up, it could come back onto both servers. If people don't want it, they don't need to vote for it

**Comment by qwertyquerty** [comment]
@crossedfall config changes

**Comment by qwertyquerty** [comment]
MRP

MIN 10 MAX 60

LRP

MIN 30 MAX 60

seem like good numbers to me crossed

**Comment by Cenrus** [comment]
I don't think Kilo is a good fit for LRP, it suffers fr being bombed/maxcapped which happens often on LRP and rarely on Sage.

**Comment by qwertyquerty** [comment]
pixel_x/y = 0 in map, and @ike709 made travis hate that

**Comment by MarkSuckerberg** [comment]
Not going to approve until all dirty vars are removed, they don't serve any purpose and are relatively easy to remove

**Comment by ike709** [comment]
I'm okay with this, if people don't want it they can just not vote for it.

I'll leave this in Crossed's hands to merge when he does the config changes.

**Comment by Fronsis** [comment]
So i don't know much about Kilo on /tg/ How much different is from the one we used to have? how many of the big important issues got solved?

**Comment by MarkSuckerberg** [comment]
aaaaa
aaaaa2
????

give me a sec, I need to check this out locally, I guess

**Comment by Cenrus** [comment]
the big issues don't matter on MRP because antagonists don't bomb the station or delaminate the engine.

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]
## Glaring issues

fakedisc
Get rid of this, it's unneeded and can be used to get illegal tech really easily

bad
I hate maps that do this

simplemobs1
simplemobs2
simplemobs3
none of these seem to be varedited to be weaker than normal, and anyways I just wish they were gone forever.

bad
I don't even

this is still an issue
Unless you've tested it, the turrets still can shoot into the bridge(!!!)

for the love of god
Just put an rwall instead of one of these doors.

BAD!
Put a wall there

I hate this
I agree with minius on this one, why even have a door there if there's a shutter to block it.

AGAGAHGHAGGGAHG
Simplemobs in maint everywhere, and a free gun (albeit an improvised one) guaranteed to spawn.

## TL;DR
In conclusion, ~~I don't really want this map back~~ the map still needs various fixes, some layout changes I haven't mentioned here since they should be done in a separate PR, and the biggest fix we did that actually helped (removing nearly every simplemob from maint and weakening external ones) were undone. I sincerely appreciate the effort done so far, but it needs much more to be considered to be readded to rotation.

**Comment by Fronsis** [comment]
I agree with Mark, i remember the fixes Mark, Prophed and some other people did that can be found in the original PR of our Kilo that should be migrated towards this /TG/ variant

While i agree yeah, on MRP it's less likely to have a bomb everysecond that causes the SM to take 1/5 of the stations the problems Mark marked (heh) are really important, besides in our version of Kilo before it was removed we had some QoL improvements and Artistics one that people really liked.

And has the pipe problem has been adressed at all? While i'm no coder or mapper i'm trying to express through my comment the main concerns people had with this map that were pointed out thousands of times, Kilo might be a good variation for MRP but there's still many things to do even if it's only for mrp.

**Comment by Cenrus** [comment]
I'm in agreement with you on those changes, and I think the map should be test merged into Sage for while to see if people like it or not.

**Comment by Cenrus** [comment]
turrets have been tested and aren't shooting at anything on the bridge any more. The grenade in the Janitor closet is basically a huge cleaning grenade, so I'll keep it in because it can't be used to unlock illegal technology.

**Comment by Toxici11i** [comment]
fuck no keep kilo deleted

**Comment by ike709** [comment]
I'm leaving this entirely in mark's hands.

**Comment by ike709** [comment]
Conflicts.

Also reminder to @MarkSuckerberg that this PR is in your hands.

**Comment by Emotions211** [comment]
I've finished most of the port. Its up to Mark if I remove the last of the simple mobs and the improvised shotgun. It should be noted that what I've left is, 
1. the improvised gun, its in a safe as it is which takes 3 c4 to crack or a lot of time with a stethoscope.
2. The two spiders left in maint have reduced health (unlike their originals which still had 150 on the poison spider).
3. The Hivelords are easy enough to deal with on the auxiliary dock, if they NEED to be removed they can be removed.

If someone really wants a good shotgun, they can steal the bartenders double barreled shotgun, besides, security should arrest anyone that acquires it as is. The security vest in maint has been removed.
The fighting pit in maint now contains 4 snakes.

I don't know if I should make a new PR or what to do with the completed map though.

**Comment by Cenrus** [comment]
updated to be included in the sage config file

**Comment by ike709** [comment]
@MarkSuckerberg

**Comment by MarkSuckerberg** [comment]
Apologies, apologies, I still don't support it, but the map is indeed playable. I still wish the AI core was more secure, but it's better.

**Review by qwertyquerty — APPROVED** [review]

**Comment by qwertyquerty** [comment]
Ready for testmerge @Crossedfall configgys

**Comment by qwertyquerty** [comment]
WAIT, can we testmerge config changes?

**Review by Crossedfall — APPROVED** [review]
As far as the config stuff goes this looks fine. I am trusting the others with their map reviews.

**Comment by Cenrus** [comment]
it's ready.

**Comment by Crossedfall** [comment]
Player feedback from the tests: Mobs (specifically the 'mooks') deal too much damage and there's a bit of a sensory overload with the tiles. The later of which I think is fine, since it would require a massive overhaul to even start addressing. The former seems easy enough, we could var edit the mobs to knock down their damage a little more. I don't personally see any problem with that.

 Otherwise players seemed to enjoy the maps return.

**Comment by Cenrus** [comment]
I replaced them with some ssssnakes instead, which only attack you if you're the instigator

**Comment by Crossedfall** [comment]
I think that should be good. If there are further problems we can address them in post.
